prosody/Dockerfile

```diff
@@ -64,9 +64,8 @@ RUN wget -qO /etc/apt/trusted.gpg.d/prosody.gpg https://prosody.im/files/prosody
     rm -rf prosody-mod-auth-matrix-user-verification-$VERSION_MATRIX_USER_VERIFICATION_SERVICE_PLUGIN v$VERSION_MATRIX_USER_VERIFICATION_SERVICE_PLUGIN.tar.gz && \
     wget -q https://github.com/jitsi-contrib/prosody-plugins/archive/refs/tags/v$VERSION_JITSI_CONTRIB_PROSODY_PLUGINS.tar.gz && \
     tar -xf v$VERSION_JITSI_CONTRIB_PROSODY_PLUGINS.tar.gz && \
-    mv prosody-plugins-$VERSION_JITSI_CONTRIB_PROSODY_PLUGINS/auth_hybrid_matrix_token/mod_auth_hybrid_matrix_token.lua /prosody-plugins && \
-    mv prosody-plugins-$VERSION_JITSI_CONTRIB_PROSODY_PLUGINS/auth_hybrid_matrix_token/mod_matrix_affiliation.lua /prosody-plugins && \
-    mv prosody-plugins-$VERSION_JITSI_CONTRIB_PROSODY_PLUGINS/auth_hybrid_matrix_token/mod_matrix_lobby_bypass.lua /prosody-plugins && \
+    mkdir /prosody-plugins-contrib && \
+    find prosody-plugins-$VERSION_JITSI_CONTRIB_PROSODY_PLUGINS -name *.lua -exec cp {} /prosody-plugins-contrib \; && \
```

Comment:
I have some concern that this may skip certain files that may be needed, such as firewall helpers or other non *.lua files.  Is there a reason we aren't just expanding the contrib plugins into this directory?  

I ask this without having studied the layout of this tar.gz, but I do recall with other prosody modules that some are just one lua file, and others are a whole directory, and that prosody supports both.

Reply:
The tarballs content looks like this:
 
```
prosody-plugins-20240117/
├── access_token
│   ├── docs
│   │   └── rsa.md
│   ├── mod_access_token.lua
│   └── README.md
├── auth_hybrid_matrix_token
│   ├── mod_auth_hybrid_matrix_token.lua
│   ├── mod_matrix_affiliation.lua
│   ├── mod_matrix_lobby_bypass.lua
│   └── README.md
├── event_sync
│   ├── mod_event_sync_component.lua
│   └── README.md
├── frozen_nick
│   ├── mod_frozen_nick.lua
│   └── README.md
├── images
│   └── nordeck.png
├── jibri_autostart
│   ├── mod_jibri_autostart.lua
│   └── README.md
├── LICENSE
├── lobby_autostart
│   ├── mod_lobby_autostart.lua
│   ├── README_legacy.md
│   └── README.md
├── per_room_max_occupants
│   ├── mod_per_room_max_occupants.lua
│   └── README.md
├── README.md
├── secure_domain_lobby_bypass
│   ├── mod_secure_domain_lobby_bypass.lua
│   └── README.md
├── time_restricted
│   ├── mod_time_restricted.lua
│   └── README.md
├── token_affiliation
│   ├── mod_token_affiliation.lua
│   └── README.md
├── token_lobby_bypass
│   ├── mod_token_lobby_bypass.lua
│   └── README.md
├── token_lobby_ondemand
│   ├── mod_token_lobby_ondemand.lua
│   └── README.md
├── token_no_wildcard
│   ├── mod_token_no_wildcard.lua
│   └── README.md
├── token_owner_party
│   ├── mod_token_owner_party.lua
│   └── README.md
└── token_security_ondemand
    ├── mod_token_security_ondemand.lua
    └── README.md
```

To me it looks like as copying just the *.lua file would be fine.. at least at the moment ;)

Reply:
But sure.. if prosody is able to load a directory structure just copying the folders would be better i think :+1: 

I did not find anything about that in https://prosody.im/doc/plugins_directory do you have any further information about loading plugins in subfolders?

fyi: i will be afk for ~ 1 week - but i will continue here after coming back :)

Reply:
Indeed, this directory-based scheme is already present for some modules, like mod_firewall here: https://github.com/jitsi/jitsi-meet/tree/master/resources/prosody-plugins

I think we should just plan to expand the tar.gz into the folder and leave the subfolder structure and test.  I expect it should Just Work (TM) but that expectation has led me astray as many times as it's proved sound.  Enjoy your time afk.